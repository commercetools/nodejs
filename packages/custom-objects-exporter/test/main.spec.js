import add from '../src/main'

describe('::add', () => {
  test('should add two numbers', () => {
    expect(add(3, 5)).toBe(8)
  })
})
