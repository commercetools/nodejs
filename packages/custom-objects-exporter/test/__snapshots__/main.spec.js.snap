// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomObjectsExporter ::buildRequest should build request 1`] = `
Object {
  "method": "GET",
  "uri": "/test-project-key/custom-objects",
}
`;

exports[`CustomObjectsExporter ::buildUri should build default uri 1`] = `"/test-project-key/custom-objects"`;

exports[`CustomObjectsExporter ::buildUri should build where/predicate uri 1`] = `"/test-project-key/custom-objects?where=key%3DcopperKey"`;

exports[`CustomObjectsExporter ::constructor should throw error if no \`apiConfig\` in \`options\` parameter 1`] = `"The constructor must be passed an \`apiConfig\` object"`;

exports[`CustomObjectsExporter ::run should write to outputStream 1`] = `
"[
{\\"foo1\\":\\"bar1\\",\\"key\\":\\"copperKey\\"}
,
{\\"foo2\\":\\"bar2\\"}
]
"
`;
